openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -97,12 +101,6 @@ def _recurring_nudge_schedules_for_hour(target_hour, org_list, exclude_orgs=Fals
         first_schedule__lt=target_hour + datetime.timedelta(minutes=60)
     )
 
-    if org_list is not None:
```

Comment:
Wait, so we're no longer using the `org_list` parameter at all?

Reply:
We are, but only to filter enrollments, not users:

https://github.com/edx/edx-platform/pull/16061/files#diff-0f5f46e92e6b2577b87433b332159d92R113